### ISSUE TITLE
passwords are not masked from Debezium JDBC config 

### ISSUE THREAD

**Issue by subkanthi** [Altinity/clickhouse-sink-connector#640]
```
`
clickhouse-sink-connector-lt_1  | [Sink connector Debezium Event Thread] INFO io.debezium.storage.jdbc.JdbcCommonConfig - Configuration for 'JdbcOffsetBackingStoreConfig' with prefix 'offset.storage.': {jdbc.password=root, jdbc.offset.table.delete=select * from %s, jdbc.user=root, jdbc.offset.table.ddl=CREATE TABLE if not exists %s ( `id` String, `offset_key` String, `offset_val` String, `record_insert_ts` DateTime, `record_insert_seq` UInt64, `_version` UInt64 MATERIALIZED toUnixTimestamp64Nano(now64(9)) ) ENGINE = ReplacingMergeTree(_version) ORDER BY offset_key SETTINGS index_granularity = 8192, jdbc.offset.table.name=altinity_sink_connector.replica_source_info, jdbc.url=jdbc:clickhouse://clickhouse:8123/altinity_sink_connector, jdbc.offset.table.select=SELECT id, offset_key, offset_val FROM %s FINAL ORDER BY record_insert_ts, record_insert_seq, file.filename=}

```
```

**Comment by subkanthi** [comment]
This is fixed in Debezium 2.7

**Comment by subkanthi** [comment]
This password should be masked
`jdbc.password=root`
